lib/CodeGen/CGExpr.cpp

```diff
@@ -4306,3 +4310,41 @@ RValue CodeGenFunction::EmitPseudoObjectRValue(const PseudoObjectExpr *E,
 LValue CodeGenFunction::EmitPseudoObjectLValue(const PseudoObjectExpr *E) {
   return emitPseudoObjectExpr(*this, E, true, AggValueSlot::ignored()).LV;
 }
+
+llvm::Value *CodeGenFunction::EmitBoundsCast(CastExpr *CE) {
+  Expr *E = CE->getSubExpr();
+  QualType DestTy = CE->getType();
+  CastKind Kind = CE->getCastKind();
+  // Bounds casting has two functionalities
+  // - code generation for explicit type casting
+  // - code generation for dynamic check, dynamic_bounds_cast only
```

Comment:
Could you update this comment?  I believe the dynamic_check expressions below need to be updated.